src/browser.ts

```diff
@@ -1,5 +1,5 @@
 /** Defines client/browser-specific variables that do not change over the course of a session. */
-
+import 'react-native-get-random-values'
```

Comment:
we import this in quite a few places, should probably only be once

Reply:
I removed the one on the app entry, the others are pretty much needed.